website/content/api-docs/auth/jwt.mdx

```diff
@@ -134,11 +134,11 @@ entities attempting to login. At least one of the bound values must be set.
 - `bound_subject` `(string: <optional>)` - If set, requires that the `sub`
   claim matches this value.
 - `bound_claims` `(map: <optional>)` - If set, a map of claims (keys) to match against respective claim values (values).
-  The expected value may be a single string or a list of strings. The interpretation of the bound
+  Each expected value may be a string, integer, boolean or a list of strings. The interpretation of the bound
```

Comment:
Here I left `a list of strings` instead of any other list, e.g. a list of integers. The reason is that the logic in the plugin needs additional refactoring to make it work and I don't know if there is a use case for it as of now. It wasn't difficult to add support for `integers`, so we kept the change minimal.

Reply:
Thanks, makes sense to me!